CRITICAL - 2025-04-18 02:13:07 --> ErrorException: Undefined variable $uri
[Method: GET, Route: admin/facility]
in APPPATH\Views\shared\dashboard_footer.php on line 32.
 1 APPPATH\Views\shared\dashboard_footer.php(32): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $uri', 'C:\\xampp\\htdocs\\207\\project_code\\wazte\\app\\Views\\shared\\dashboard_footer.php', 32)
 2 SYSTEMPATH\View\View.php(224): include('C:\\xampp\\htdocs\\207\\project_code\\wazte\\app\\Views\\shared\\dashboard_footer.php')
 3 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1173): CodeIgniter\View\View->render('shared/dashboard_footer', [], true)
 5 APPPATH\Controllers\Admin.php(95): view('shared/dashboard_footer', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Admin->facility()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-04-18 12:49:32 --> ErrorException: Undefined variable $googlekey
[Method: GET, Route: facilitator/facility]
in APPPATH\Views\shared\dashboard_header.php on line 33.
 1 APPPATH\Views\shared\dashboard_header.php(33): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $googlekey', 'C:\\xampp\\htdocs\\207\\project_code\\wazte\\app\\Views\\shared\\dashboard_header.php', 33)
 2 SYSTEMPATH\View\View.php(224): include('C:\\xampp\\htdocs\\207\\project_code\\wazte\\app\\Views\\shared\\dashboard_header.php')
 3 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1173): CodeIgniter\View\View->render('shared/dashboard_header', [], true)
 5 APPPATH\Controllers\Facilitator.php(86): view('shared/dashboard_header')
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Facilitator->facility()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Facilitator))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-04-18 20:34:56 --> ErrorException: Undefined variable $googlekey
[Method: GET, Route: facilitator/facility]
in APPPATH\Views\shared\dashboard_header.php on line 42.
 1 APPPATH\Views\shared\dashboard_header.php(42): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $googlekey', 'C:\\xampp\\htdocs\\207\\project_code\\wazte\\app\\Views\\shared\\dashboard_header.php', 42)
 2 SYSTEMPATH\View\View.php(224): include('C:\\xampp\\htdocs\\207\\project_code\\wazte\\app\\Views\\shared\\dashboard_header.php')
 3 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1173): CodeIgniter\View\View->render('shared/dashboard_header', [], true)
 5 APPPATH\Controllers\Facilitator.php(86): view('shared/dashboard_header')
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Facilitator->facility()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Facilitator))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-04-18 20:40:11 --> ErrorException: Undefined array key "tags_ID"
[Method: GET, Route: admin/facility]
in APPPATH\Views\admin\facility\index.php on line 85.
 1 APPPATH\Views\admin\facility\index.php(85): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "tags_ID"', 'C:\\xampp\\htdocs\\207\\project_code\\wazte\\app\\Views\\admin\\facility\\index.php', 85)
 2 SYSTEMPATH\View\View.php(224): include('C:\\xampp\\htdocs\\207\\project_code\\wazte\\app\\Views\\admin\\facility\\index.php')
 3 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1173): CodeIgniter\View\View->render('admin/facility/index', [], true)
 5 APPPATH\Controllers\Admin.php(103): view('admin/facility/index', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Admin->facility()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-04-18 20:42:20 --> mysqli_sql_exception: Unknown column 'tagas_id' in 'field list' in C:\xampp\htdocs\207\project_code\wazte\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\207\project_code\wazte\system\Database\MySQLi\Connection.php(327): mysqli->query('SELECT `tagas_i...', 0)
#1 C:\xampp\htdocs\207\project_code\wazte\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `tagas_i...')
#2 C:\xampp\htdocs\207\project_code\wazte\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `tagas_i...')
#3 C:\xampp\htdocs\207\project_code\wazte\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT `tagas_i...', Array, false)
#4 C:\xampp\htdocs\207\project_code\wazte\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\207\project_code\wazte\system\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\207\project_code\wazte\app\Models\MaterialModel.php(15): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\207\project_code\wazte\app\Controllers\Admin.php(92): App\Models\MaterialModel->getMaterials()
#8 C:\xampp\htdocs\207\project_code\wazte\system\CodeIgniter.php(933): App\Controllers\Admin->facility()
#9 C:\xampp\htdocs\207\project_code\wazte\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
#10 C:\xampp\htdocs\207\project_code\wazte\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\207\project_code\wazte\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\207\project_code\wazte\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\207\project_code\wazte\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2025-04-18 20:42:20 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'tagas_id' in 'field list'
[Method: GET, Route: admin/facility]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT `tagas_id` as `id`, `Material` as `name`, `icon`
FROM `tags`', [], false)
 2 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Models\MaterialModel.php(15): CodeIgniter\BaseModel->findAll()
 5 APPPATH\Controllers\Admin.php(92): App\Models\MaterialModel->getMaterials()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Admin->facility()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-04-18 20:42:20 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'tagas_id' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `tagas_id` as `id`, `Material` as `name`, `icon`
FROM `tags`')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `tagas_id` as `id`, `Material` as `name`, `icon`
FROM `tags`')
 3 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT `tagas_id` as `id`, `Material` as `name`, `icon`
FROM `tags`', [], false)
 4 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
 6 APPPATH\Models\MaterialModel.php(15): CodeIgniter\BaseModel->findAll()
 7 APPPATH\Controllers\Admin.php(92): App\Models\MaterialModel->getMaterials()
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Admin->facility()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-04-18 20:42:20 --> [Caused by] mysqli_sql_exception: Unknown column 'tagas_id' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('SELECT `tagas_id` as `id`, `Material` as `name`, `icon`
FROM `tags`', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `tagas_id` as `id`, `Material` as `name`, `icon`
FROM `tags`')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `tagas_id` as `id`, `Material` as `name`, `icon`
FROM `tags`')
 4 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT `tagas_id` as `id`, `Material` as `name`, `icon`
FROM `tags`', [], false)
 5 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
 7 APPPATH\Models\MaterialModel.php(15): CodeIgniter\BaseModel->findAll()
 8 APPPATH\Controllers\Admin.php(92): App\Models\MaterialModel->getMaterials()
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Admin->facility()
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-04-18 21:46:37 --> mysqli_sql_exception: Column 'AddedBy' cannot be null in C:\xampp\htdocs\207\project_code\wazte\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\207\project_code\wazte\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `fa...', 0)
#1 C:\xampp\htdocs\207\project_code\wazte\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `fa...')
#2 C:\xampp\htdocs\207\project_code\wazte\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `fa...')
#3 C:\xampp\htdocs\207\project_code\wazte\system\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `fa...', Array, false)
#4 C:\xampp\htdocs\207\project_code\wazte\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\207\project_code\wazte\system\BaseModel.php(839): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\207\project_code\wazte\system\Model.php(800): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\207\project_code\wazte\app\Controllers\Facility.php(50): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\207\project_code\wazte\system\CodeIgniter.php(933): App\Controllers\Facility->createNewFacility()
#9 C:\xampp\htdocs\207\project_code\wazte\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Facility))
#10 C:\xampp\htdocs\207\project_code\wazte\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\207\project_code\wazte\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\207\project_code\wazte\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\207\project_code\wazte\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2025-04-18 21:46:37 --> CodeIgniter\Database\Exceptions\DatabaseException: Column 'AddedBy' cannot be null
[Method: POST, Route: facility/saveNewfacility]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `facilities` (`locationID`, `dateadded`, `AddedBy`, `contactNo`, `name`, `Description`, `BusinessHours`) VALUES (:locationID:, :dateadded:, :AddedBy:, :contactNo:, :name:, :Description:, :BusinessHours:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(839): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\Facility.php(50): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Facility->createNewFacility()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Facility))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-04-18 21:46:37 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Column 'AddedBy' cannot be null
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `facilities` (`locationID`, `dateadded`, `AddedBy`, `contactNo`, `name`, `Description`, `BusinessHours`) VALUES (1, \'2025-04-18 21:46:37\', NULL, \'09228550341\', \'Test Facility\', \'test\', \'9am-10pm\')')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `facilities` (`locationID`, `dateadded`, `AddedBy`, `contactNo`, `name`, `Description`, `BusinessHours`) VALUES (1, \'2025-04-18 21:46:37\', NULL, \'09228550341\', \'Test Facility\', \'test\', \'9am-10pm\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `facilities` (`locationID`, `dateadded`, `AddedBy`, `contactNo`, `name`, `Description`, `BusinessHours`) VALUES (:locationID:, :dateadded:, :AddedBy:, :contactNo:, :name:, :Description:, :BusinessHours:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(839): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\Facility.php(50): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Facility->createNewFacility()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Facility))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-04-18 21:46:37 --> [Caused by] mysqli_sql_exception: Column 'AddedBy' cannot be null
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `facilities` (`locationID`, `dateadded`, `AddedBy`, `contactNo`, `name`, `Description`, `BusinessHours`) VALUES (1, \'2025-04-18 21:46:37\', NULL, \'09228550341\', \'Test Facility\', \'test\', \'9am-10pm\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `facilities` (`locationID`, `dateadded`, `AddedBy`, `contactNo`, `name`, `Description`, `BusinessHours`) VALUES (1, \'2025-04-18 21:46:37\', NULL, \'09228550341\', \'Test Facility\', \'test\', \'9am-10pm\')')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `facilities` (`locationID`, `dateadded`, `AddedBy`, `contactNo`, `name`, `Description`, `BusinessHours`) VALUES (1, \'2025-04-18 21:46:37\', NULL, \'09228550341\', \'Test Facility\', \'test\', \'9am-10pm\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `facilities` (`locationID`, `dateadded`, `AddedBy`, `contactNo`, `name`, `Description`, `BusinessHours`) VALUES (:locationID:, :dateadded:, :AddedBy:, :contactNo:, :name:, :Description:, :BusinessHours:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(839): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\Facility.php(50): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Facility->createNewFacility()
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Facility))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-04-18 21:49:23 --> Error: Class "App\Controllers\Services" not found
[Method: POST, Route: facility/saveNewfacility]
in APPPATH\Controllers\Facility.php on line 13.
 1 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Facility->createNewFacility()
 2 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Facility))
 3 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-04-18 21:51:42 --> mysqli_sql_exception: Column 'AddedBy' cannot be null in C:\xampp\htdocs\207\project_code\wazte\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\207\project_code\wazte\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `fa...', 0)
#1 C:\xampp\htdocs\207\project_code\wazte\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `fa...')
#2 C:\xampp\htdocs\207\project_code\wazte\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `fa...')
#3 C:\xampp\htdocs\207\project_code\wazte\system\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `fa...', Array, false)
#4 C:\xampp\htdocs\207\project_code\wazte\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\207\project_code\wazte\system\BaseModel.php(839): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\207\project_code\wazte\system\Model.php(800): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\207\project_code\wazte\app\Controllers\Facility.php(52): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\207\project_code\wazte\system\CodeIgniter.php(933): App\Controllers\Facility->createNewFacility()
#9 C:\xampp\htdocs\207\project_code\wazte\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Facility))
#10 C:\xampp\htdocs\207\project_code\wazte\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\207\project_code\wazte\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\207\project_code\wazte\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\207\project_code\wazte\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2025-04-18 21:51:42 --> CodeIgniter\Database\Exceptions\DatabaseException: Column 'AddedBy' cannot be null
[Method: POST, Route: facility/saveNewfacility]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `facilities` (`locationID`, `dateadded`, `AddedBy`, `contactNo`, `name`, `Description`, `BusinessHours`) VALUES (:locationID:, :dateadded:, :AddedBy:, :contactNo:, :name:, :Description:, :BusinessHours:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(839): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\Facility.php(52): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Facility->createNewFacility()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Facility))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-04-18 21:51:42 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Column 'AddedBy' cannot be null
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `facilities` (`locationID`, `dateadded`, `AddedBy`, `contactNo`, `name`, `Description`, `BusinessHours`) VALUES (2, \'2025-04-18 21:51:42\', NULL, \'09228550341\', \'Test Facility\', \'test\', \'9am-10pm\')')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `facilities` (`locationID`, `dateadded`, `AddedBy`, `contactNo`, `name`, `Description`, `BusinessHours`) VALUES (2, \'2025-04-18 21:51:42\', NULL, \'09228550341\', \'Test Facility\', \'test\', \'9am-10pm\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `facilities` (`locationID`, `dateadded`, `AddedBy`, `contactNo`, `name`, `Description`, `BusinessHours`) VALUES (:locationID:, :dateadded:, :AddedBy:, :contactNo:, :name:, :Description:, :BusinessHours:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(839): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\Facility.php(52): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Facility->createNewFacility()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Facility))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-04-18 21:51:42 --> [Caused by] mysqli_sql_exception: Column 'AddedBy' cannot be null
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `facilities` (`locationID`, `dateadded`, `AddedBy`, `contactNo`, `name`, `Description`, `BusinessHours`) VALUES (2, \'2025-04-18 21:51:42\', NULL, \'09228550341\', \'Test Facility\', \'test\', \'9am-10pm\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `facilities` (`locationID`, `dateadded`, `AddedBy`, `contactNo`, `name`, `Description`, `BusinessHours`) VALUES (2, \'2025-04-18 21:51:42\', NULL, \'09228550341\', \'Test Facility\', \'test\', \'9am-10pm\')')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `facilities` (`locationID`, `dateadded`, `AddedBy`, `contactNo`, `name`, `Description`, `BusinessHours`) VALUES (2, \'2025-04-18 21:51:42\', NULL, \'09228550341\', \'Test Facility\', \'test\', \'9am-10pm\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `facilities` (`locationID`, `dateadded`, `AddedBy`, `contactNo`, `name`, `Description`, `BusinessHours`) VALUES (:locationID:, :dateadded:, :AddedBy:, :contactNo:, :name:, :Description:, :BusinessHours:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(839): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\Facility.php(52): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Facility->createNewFacility()
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Facility))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
